pool/message.go

```diff
@@ -349,9 +349,14 @@ func ParseSubscribeResponse(resp *Response) (string, string, string, uint64, err
 // SetDifficultyNotification creates a set difficulty notification message.
 func SetDifficultyNotification(difficulty *big.Rat) *Request {
 	diff, _ := difficulty.Float64()
+	iDiff := uint64(diff)
```

Comment:
Could you use a more descriptive name instead of `iDiff` here?

Reply:
Done, also added a little comment to explain.